### PR TITLE
Add upgrade support

### PR DESCRIPTION
**Pull request by hrpatel**

### CODE REVIEW

**Comment by antevens** [comment]
+1

**Review by antevens — DISMISSED** [review]
+1

File: nginx-php-moodle/bin/configure-moodle.sh [hunk 1]
```diff
@@ -67,20 +51,67 @@ if wait_for_success "/etc/init.d/php7.2-fpm status"; then
...
+            color_echo green "'config.php' matches backup. Nothing to do"
+            return
+        fi
+    else
+        # Most likely a new container
+        color_echo green "'config.php' does not exist, restoring from backup."
+    fi
+
+    color_echo yellow "Restoring Moodle configuration from backup"
+    cp -fv /opt/moodle/backup/config.php "/opt/moodle/moodle-${MOODLE_VERSION}/config.php"
```
Comment (mroote) [inline nginx-php-moodle/bin/configure-moodle.sh:86]: Do we need the force option here?

**Comment by hrpatel** [comment]
Sorry guys. Had to commit a change to get CI to pass - please review again.

**Comment by antevens** [comment]
I don't think you need another review/approval :D

File: nginx-php-moodle/bin/configure-moodle.sh (hunk 1 above)
Comment (hrpatel) [reply nginx-php-moodle/bin/configure-moodle.sh:86]: Yeah, if the target file exists and its different a warning is logged and the file is overwritten

**Comment by hrpatel** [comment]
@antevens I configured GH to revoke an approval if commits are added after an approval

**Review by antevens — DISMISSED** [review]

**Review by mroote — APPROVED** [review]
+1

**Pull request merged**